community/team.html.haml

```diff
@@ -0,0 +1,72 @@
+---
+layout: community-standard
+title: Hibernate Team
+---
+
+%h2{:id => "leaders"} Project Leaders
+
+.ui.icon.message
+  %i.icon.id.badge
+  .content
+    This is a list of all
+    %a(href="/community/governance/#roles")
+      Hibernate Project Leaders
+    \.
+
+.contributor-list.leader-list
+  - site.governance.leaders.each do |login, project_names|
+    - project_names_joined = project_names.map { |key| site.projects[key].name } .join( ', ' )
+    .ui.card.contributor{:"data-github-login" => login, :"data-project-names" => project_names_joined}
+      .content
+        .header
+          %a{:href => "https://github.com/#{login}"}
+            #{login}
+      .extra.content
+        Leader of #{project_names_joined}.
+
+%h2{:id => "members"} Members
+
+.ui.icon.message
+  %i.icon.id.badge
+  .content
+    This is a list of all
+    %a(href="/community/governance/#roles")
+      Hibernate Members
+    who decided to make their membership public on GitHub.
```

Comment:
Oddly, some very active members have made their membership private... Maybe by mistake?

If you read this, please check your membership's visibility. See https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-membership-in-organizations/publicizing-or-hiding-organization-membership